driver-puppeteer/index.js

```diff
@@ -7,19 +7,19 @@ exports.mochifyDriver = mochifyDriver;
 const default_url = `file:${__dirname}/index.html`;
 
 async function mochifyDriver(options = {}) {
-  const stderr = options.stderr || process.stderr;
+  const { stderr = process.stderr, ...launch_options } = options;
+
+  launch_options.args = [
+    '--allow-insecure-localhost',
+    '--disable-dev-shm-usage',
+    ...(launch_options.args || [])
+  ];
 
   const browser = await driver.launch({
-    ignoreHTTPSErrors: true
-    /*
-    args: [
-      '--allow-insecure-localhost',
-      '--disable-dev-shm-usage',
-      '--no-sandbox',
-      '--disable-setuid-sandbox',
-      '--disable-web-security'
-    ]
-    */
+    ignoreHTTPSErrors: true,
+    // Workaround for https://github.com/puppeteer/puppeteer/issues/6957
+    executablePath: process.env.PUPPETEER_EXECUTABLE_PATH,
+    ...launch_options
```

Comment:
Does this allow to override `executablePath` via the `launch_options`?

Reply:
Yup, that would still work:

```console
➜  ~ FOO_BAR=some_value node
Welcome to Node.js v14.15.5.
Type ".help" for more information.
> const opts = { foo_bar: 'other_value' }
undefined
> { foo_bar: process.env.FOO_BAR, ...opts }
{ foo_bar: 'other_value' }
```

We can also use `Object.assign` though if that's easier to understand?